### ISSUE TITLE
test

### ISSUE THREAD

**Issue by SHIJILIANG01** [dengyecode/hourglassattention#10]
Hello, I have the following problem when testing, the results show that the GPU memory is not enough caused by this reason is too big model? I used an RTX 3090 (24GB) and tested it on 3000 images.

RuntimeError: CUDA out of memory. Tried to allocate 1.50 GiB (GPU 0; 23.69 GiB total capacity; 21.57 GiB already allocated; 422.94 MiB free; 21.60 GiB reserved in total by PyTorch)

**Comment by SHIJILIANG01** [comment]
I did this by adding, with torch.no_grad(): during test, before the dataloader loop, I don't know if I'm doing it right

**Comment by SHIJILIANG01** [comment]
I don't use the --continue_train parameter in my tests, does this work?

**Comment by dengyecode** [comment]
Yes, it is not necessary to use "--continue_train". In addition, please make sure the batch_size is 1 during the test.

**Comment by NoDojuanwang** [comment]
When I ran the code, I encountered 'test. py: error: unrecognized arguments: -- image_file'
”May I ask how to solve this problem? I have collected a lot of information but cannot solve it. Thank you very much

**Comment by SHIJILIANG01** [comment]
You need to change -- image_file to -- img_file




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "dengyecode/hourglassattention"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年6月25日(星期天) 上午10:09
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [dengyecode/hourglassattention] test (Issue #10)





 
When I ran the code, I encountered 'test. py: error: unrecognized arguments: -- image_file'
 ”May I ask how to solve this problem? I have collected a lot of information but cannot solve it. Thank you very much

**Comment by Sheeran2000** [comment]
> 您需要将 --image_file 更改为 --img_file
> [……](#)
> ------------------ 原始邮件 -------------------- 发件人: "dengyecode/hourglassattention "* **@***.***>;发送时间：2023年6月25日(周日)上午10:09 ***@***.***>;***@***.* *****@***.***>；主题：回复：[dengyecode/hourglassattention] test (Issue #10 ) 当我运行代码时，我遇到了 'test.py: error: 无法识别的参数: -- image_file' ” 请问如何解决这个问题？我收集了很多信息，但无法解决。非常感谢 - 直接回复此电子邮件、在 GitHub 上查看或取消订阅。您收到此消息是因为您创作了该主题。消息ID：***@***.***>

您完整的复现出来了吗 指标相差大吗
